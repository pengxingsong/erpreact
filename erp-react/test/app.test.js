import React from "react";
import { shallow, mount, render } from "enzyme";
import toJson from "enzyme-to-json"; 
import App from '../src/App';

test('test:/src/app.render', () => {
  const Guide_render = render(
    <App /> //直接使用store将引入的store传给待测组件
  );
  console.log(toJson(Guide_render))
  //expect(toJson(Guide_render)).toMatchSnapshot();  //生成快照
});