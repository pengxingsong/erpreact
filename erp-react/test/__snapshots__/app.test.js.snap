exports[`test test:/src/app.render 1`] = `
<div
  class="App">
  <div
    class="App-header">
    <h2>
      Welcome to React
    </h2>
  </div>
  <p
    class="App-intro">
    To get started, edit 
    <code>
      src/App.js
    </code>
     and save to reload.
  </p>
</div>
`;
